src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/ActiveWorkspaceProjectContextHost.cs

```diff
@@ -16,18 +17,23 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
     internal class ActiveWorkspaceProjectContextHost : IActiveWorkspaceProjectContextHost
     {
         private readonly ActiveConfiguredProject<IWorkspaceProjectContextHost> _activeHost;
-        private readonly ActiveConfiguredProject<IConfiguredProjectImplicitActivationTracking> _activeConfiguredProject;
+        private readonly IUnconfiguredProjectTasksService _tasksService;
```

Comment:
Did this change purely because it wasn't necessary, or what there some stronger reason?